kamelets/jms-ibm-mq-sink.kamelet.yaml

```diff
@@ -0,0 +1,124 @@
+    camel.apache.org/kamelet.support.level: "Preview"
+    camel.apache.org/provider: "Apache Software Foundation"
+    camel.apache.org/catalog.version: "main-SNAPSHOT"
+    camel.apache.org/kamelet.group: "JMS"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+    camel.apache.org/requires.runtime: camel-k
+spec:
+  definition:
+    title: "JMS - IBM MQ Kamelet Sink"
+    description: |-
+      A Kamelet that can produce events to an IBM MQ message queue using JMS.
+
+      For licensing reasons, the IBM MQ Java library is not declared in this Kamelet.
+
+      For this Kamelet to work, the IBM MQ Java library MUST be added to the Kamelet, int the `dependencies` section, add the dependency `"mvn:com.ibm.mq:com.ibm.mq.allclient:9.2.5.0"`.
```

Comment:
Should be explained similarly too:
https://github.com/apache/camel-kamelets/blob/main/kamelets/sqlserver-sink.kamelet.yaml#L35-#L37